src/measure_theory/measure/vector_measure.lean

```diff
@@ -985,6 +985,31 @@ end
 
 end absolutely_continuous
 
+/-- Twp vector measures `v` and `w` are said to be mutually singular if there exists a
+measurable set `s` such that `v s = 0` and `w sᶜ = 0`. -/
+def mutually_singular (v : vector_measure α M) (w : vector_measure α N) : Prop :=
+∃ (s : set α), measurable_set s ∧ v s = 0 ∧ w sᶜ = 0
+
+localized "infix ` ⊥ᵥ `:60 := measure_theory.vector_measure.mutually_singular" in measure_theory
+
+namespace mutually_singular
+
+variables {v : vector_measure α M} {w : vector_measure α N}
+
+lemma zero : v ⊥ᵥ (0 : vector_measure α N) :=
+⟨∅, measurable_set.empty, v.empty, rfl⟩
+
+lemma symm (h : v ⊥ᵥ w) : w ⊥ᵥ v :=
+let ⟨i, hi, hiv, hiw⟩ := h in
+  ⟨iᶜ, measurable_set.compl hi, hiw, (compl_compl i).symm ▸ hiv⟩
+
+lemma smul {R : Type*} [semiring R] [distrib_mul_action R M] [topological_space R]
```

Comment:
can you call this one `smul_left`, and have also an `smul_right` (and maybe even an `smul` saying that `c v` is singular wrt `d w`?)